packages/safe-apps-sdk/dist/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@safe-global/safe-apps-sdk",
-    "version": "7.10.0",
+    "version": "7.10.1",
```

Comment:
The version should be bumped via changeset

Reply:
You need to create a changeset and push it to the dev branch. The changeset bot will create a PR to bump the version